testing.sagebrush.master/Stepwat_Inputs/Input/rgroup.in

```diff
@@ -52,6 +52,8 @@
 #            Overrides grazingyear.
 # proportion_grazing = proportion of biomass removed due to grazing. 
 # grazingfreq_startyr = start year for grazing
+# biomass, along with transpiration, construct a simple linear relationship (grams per centimeter) used to calculate bvt.
+# transpiration, along with biomass, construct a simple linear relationship (grams per centimeter) used to calculate bvt.
 
```

Comment:
Maybe expand on units [grams per centimeter] to make obvious that "centimeter" refers to "centimeters of transpired water"?
--> e.g., [grams per centimeter of transpired water]

Reply:
thank you! addressed by 23d9f23